setup.py

```diff
@@ -45,5 +45,6 @@
         click-odoo-initdb=click_odoo_contrib.initdb:main
         click-odoo-backupdb=click_odoo_contrib.backupdb:main
         click-odoo-makepot=click_odoo_contrib.makepot:main
+        click-odoo-set=click_odoo_contrib.set:main
```

Comment:
The command name is a bit terse. I suppose it should mention what it sets, somehow.